andela/elendil-cfh
# Feature(152796599) Invite 11 other users

### What does this PR do?
Allow Users invite other 11 users

### Description of Task to be completed?

### How should this be manually tested?
 - run npm test in terminal.
- Navigate to the site, click the play game, it would pop up the select region modal, select the region of choice and play game with questions from that region
- It would pop up another modal in which you should see invite user
- Type user name or email to send Invites

### Any background context you want to provide?

### What are the relevant pivotal tracker stories?
[#152796599](https://www.pivotaltracker.com/n/projects/2127911)

Screenshots (if appropriate)
<img width="505" alt="screen shot 2017-12-11 at 12 56 50 pm" src="https://user-images.githubusercontent.com/22414567/33830675-61ff166e-de75-11e7-9e92-1f0a87e097e2.png">

<img width="508" alt="screen shot 2017-12-11 at 1 14 22 pm" src="https://user-images.githubusercontent.com/22414567/33830714-897210de-de75-11e7-9924-e71c8ab9e180.png">

<img width="458" alt="screen shot 2017-12-11 at 1 14 45 pm" src="https://user-images.githubusercontent.com/22414567/33830712-894f9824-de75-11e7-925f-a95c92d65529.png">

<img width="620" alt="screen shot 2017-12-11 at 1 19 54 pm" src="https://user-images.githubusercontent.com/22414567/33830831-0b399a56-de76-11e7-926a-f402fa8380cc.png">
